src/us49.py

```diff
@@ -0,0 +1,27 @@
+'''
+Author: Grace Miguel
+Date: 11/27/2020
+I pledge my honor that I've abided by the Stevens Honor Code.
+'''
+import sys
+import os
+#from Gedcom import Gedcom, Family
+from Gedcom import Gedcom, Family 
+#This function lists families with only daughters
+def us49(gedcom_file):
+    children = []
+    gedcom = Gedcom(gedcom_file)
+    families = gedcom.families
+    x = 0
+    for i in families:
+        for j in i.child:
```

Comment:
Similarly here and through out the rest of the PR, should try to follow a readable naming scheme

Reply:
fixed!